packages/js/test/unit/client.test.ts

```diff
@@ -194,4 +194,17 @@ describe('Axiom', () => {
     expect(response).not.toEqual('undefined');
     expect(response.matches).toHaveLength(2);
   });
+  it('should successfully handle circular JSON during ingestion with correct options', async () => {
+    const circularReferenceObject: any = { foo: 'bar', circularAttribute: null };
```

Comment:
Instead of disabling type checking by using `any` what if we use this? 🤔 

```ts
type CircularReferenceObject = { foo: string; circularAttribute: CircularReferenceObject | null };
const circularReferenceObject: CircularReferenceObject = { foo: 'bar', circularAttribute: null };
circularReferenceObject.circularAttribute = circularReferenceObject;
```